contrib/cirrus/runner.sh

```diff
@@ -43,6 +45,15 @@ _run_setup() {
     "${SKOPEO_PATH}/${SCRIPT_BASE}/runner.sh" setup
 }
 
+_run_debian_setup() {
+    msg "Installing previously downloaded/cached Docker packages"
+    ooe.sh dpkg -i \
+        $PACKAGE_DOWNLOAD_DIR/containerd.io*.deb \
+        $PACKAGE_DOWNLOAD_DIR/docker-ce*.deb
+    git remote update
+    make tools
```

Comment:
Keeping this in the Cirrus `validate_task`, and not doing that for the other ones, might speed up the other ones a tiny bit.

Arguably we should fix the `Makefile` so that it installs tools automatically on-demand, so that users of `make validate` don’t need to care.

And I guess ideally the tools could be pre-installed in the VM, though that’s definitely not blocking this PR and it can be done when/if someone has extra time.

Reply:
Good suggestion about avoiding `make tools` beyond validate, I'll do that.